### PR TITLE
feat(curriculum): Add questions about CSS libraries and frameworks

### LINKED ISSUE #56228: [HACKTOBERFEST 2024] - CSS Libraries and Frameworks Quiz
> [!IMPORTANT]
> This issue is part of our 2024 Hacktoberfest initiative. If you are interested in working on this issue, refer to our [top-level tasks](https://github.com/freeCodeCamp/freeCodeCamp/issues/56170) to see if this issue is claimed, or to claim it if it is not.

This issue is to create the CSS Libraries and Frameworks quiz.

The file you should be working on is found in `/curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks`.

**All quiz questions need to relate to one of these topics:**

- What is a CSS framework
- Advantages and disadvantages of using one
- **BASIC** examples of tailwind, bootstrap, materialize
- preprocessors like Sass, SCSS

> [!CAUTION]
> Questions which do not fall under these topics will be rejected.

You are welcome to include code blocks in your questions, but please keep them short (four lines or less).

Please do NOT open a pull request until you have completed all questions. Your quiz should have **exactly 20 questions**.

### PR DESCRIPTION
**Pull request by ReenaChidrala**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56228

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 1]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is a CSS framework? 
 
 #### --distractors--
 
-Placeholder distractor 1
+set of CSS rules
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:24]:
```suggestion
A set of CSS rules
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 2]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 2
+A JavaScript library  
 
 ---
 
-Placeholder distractor 3
+coding language
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:32]:
```suggestion
A coding language
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 3]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 #### --answer--
 
-Placeholder answer
+library for CSS styles
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:36]:
```suggestion
A library for CSS styles
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 4]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+Tailwind
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the primary purpose of the .container class in Bootstrap?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:130]:
```suggestion
What is the primary purpose of the `.container` class in Bootstrap?
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 5]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 #### --text--
 
-Placeholder question
+In Tailwind CSS, which class is used to add margin to an element?
 
 #### --distractors--
 
-Placeholder distractor 1
+m-4
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:288]:
```suggestion
`m-4`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 6]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 2
+padding-4
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:292]:
```suggestion
`padding-4`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 7]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 3
+margin-4
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:296]:
```suggestion
`margin-4`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 8]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 #### --answer--
 
-Placeholder answer
+mt-4
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:300]:
```suggestion
`mt-4`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 9]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 #### --text--
 
-Placeholder question
+What is the file extension for SCSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+.sass
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:376]:
```suggestion
`.sass`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 10]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 2
+.scsss
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:380]:
```suggestion
`.scsss`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 11]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 3
+.css
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:384]:
```suggestion
`.css`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 12]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 #### --answer--
 
-Placeholder answer
+.scss
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:388]:
```suggestion
`.scss`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 13]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 #### --text--
 
-Placeholder question
+Which Tailwind class would you use to set text color to red?
 
 #### --distractors--
 
-Placeholder distractor 1
+text-red
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:420]:
```suggestion
`text-red`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 14]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+Responsive layout
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which Tailwind class would you use to set text color to red?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:416]:
```suggestion
Which Tailwind class would you use to set the text color to red?
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 15]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 #### --distractors--
 
-Placeholder distractor 1
+text-red
 
 ---
 
-Placeholder distractor 2
+red-text
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:424]:
```suggestion
`red-text`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 16]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 3
+color-red
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:428]:
```suggestion
`color-red`
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md [hunk 17]
```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 #### --answer--
 
-Placeholder answer
+text-red-500
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-libraries-and-frameworks/66f1aeb60b11aec5abe83c2e.md:432]:
```suggestion
`text-red-500`
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Comment by gikf** [comment]
Hey @ReenaChidrala,

did you get a chance to take a look at the suggested changes?

**Comment by ReenaChidrala** [comment]
@gikf "Hi ma'am, I have made the changes as per your suggestion. Please take a look."

**Review by naomi-lgbt — APPROVED** [review]

**Comment by ReenaChidrala** [comment]
@gikf pls review my pr

**Review by gikf — APPROVED** [review]

**Comment by gikf** [comment]
Hi @ReenaChidrala. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Comment by ReenaChidrala** [comment]
@gikf 
Thank You so Much For Accepting My Pr And Form Such a Nice Comment

**Pull request merged**